script/init.sh

```diff
@@ -6,12 +6,12 @@ echo -e "Welcome to the Maji installer. This script will help you setup your new
 
 while [[ -z "$APP_PACKAGE" ]]
 do
-  read -p "Enter a package name: " APP_PACKAGE
+  read -p "Enter a package name (e.g. org.example.project): " APP_PACKAGE
 done
 
 while [[ -z "$APP_PATH" ]]
 do
-  read -p "Enter a directory: " APP_PATH
+  read -p "Enter a directory (e.g. /Users/bob/Code/project): " APP_PATH
```

Comment:
Technically, after #193, you could also pass in a relative path. Maybe change the example to `./Code/project`, `project/` or something like that?

Reply:
Good point. Initially I tried adding both an absolute and a relative path but that as too long. Just a relative path might be better indeed.